portfolio/src/main/webapp/script.js

```diff
@@ -16,16 +16,19 @@ $(document).ready(function() {
     $('#unfilter-button').css("display","none");
     handleFilterButton();
     handleUnfilterButton();
-  
+    
     $('#2048-pic').css("display","none");
     $('#tweeter-pic').css("display","none");
     $('#job-pic').css("display","none");
     $('#mmr-pic').css("display","none");
     handleClickProject();
-
+    //enable comment function in main page
```

Comment:
For comments, capitalize them and add a space between `//` and text.

`// Enable comment function in main page.`